_sass/listing.scss

```diff
@@ -74,10 +75,10 @@
 
     .img-wrapper {
       display: flex;
-      flex-basis: 100%;
       min-width: 220px;
-      max-width: 40%;
       max-height: 300px;
+      height: auto;
+      flex-shrink: 2;
 
```

Comment:
I think the images should be bigger. I really liked how they looked originally before we put in our own paragraph where the images were horizontal. Closer to the design mockup.

https://drive.google.com/drive/u/1/folders/1GiOw7knlJ31QzCf6XTqEJfLcx7VkMdEs